controller/tests/array_action/ds8k/test_array_mediator_ds8k.py

```diff
@@ -1,6 +1,6 @@
 import unittest
 from munch import Munch
-from mock import patch, NonCallableMagicMock
+from mock import patch, NonCallableMagicMock, Mock
```

Comment:
what is this for?

Reply:
removed
code scanning shold had warn i think......

Reply:
CSI-1196